### DISCUSSION TITLE
Dead Repo

### DISCUSSION THREAD

**Discussion by Javadebi** [amzn/selling-partner-api-models#3072] [General]
Hello dead Repository. If anyone is alive here, is there anyway rather than `GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA` to get my FBA fee because `GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA` is outdate and as it mentioned it may take 3 days to give updated FBA fee.

**Comment by cmlkx** [comment]
Hi @Javadebi ,

You can get the fees for a specific product by using this API:
https://developer-docs.amazon.com/sp-api/docs/product-fees-v0-use-case-guide

You'd have to call the API for each item you need the fees for, so it is not as nice as GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA.  Hopefully that helps!
